eight-day-week.php

```diff
@@ -60,6 +60,49 @@
 define( 'EDW_ARTICLE_STATUS_TAX', 'pi-article-status' );
 define( 'EDW_AJAX_NONCE_SLUG', 'edw_ajax_nonce' );
 
+/**
+ * Get the minimum version of PHP required by this plugin.
+ *
+ * @return string Minimum version required.
+ */
+function minimum_php_requirement() {
```

Comment:
Maybe add the prefix `edw_` here as we don't have a namespace in this file? Or add a namespace?

Reply:
@iamdharmesh Nice catch, updated accordingly!